content/en/agent/guide/agent-5-commands.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Agent 5 Commands
+kind: Guide
+disable_toc: false
+---
+
+## Overview
+
+This page covers Agent 5 commands. Datadog recommends installing or upgrading to Agent 7 for the latest features. For information on installing the latest version of the Agent, follow the [Agent 7 Installation Instructions][1]. For information on upgrading to Agent 7 from an earlier version, see [Upgrade to Datadog Agent v7][2].
```

Comment:
Could we just say "the latest version of the agent" instead of referencing a specific version number?

I don't know how often new versions of the agent roll out, so I don't know how much of a headache this is.

Reply:
I went back and forth on this one. I have an idea though